src/gui/sdl_mapper.cpp

```diff
@@ -266,9 +266,6 @@ class CBind {
 	}
 
 	void ActivateBind(Bits _value,bool ev_trigger,bool skip_action=false) {
-		if (!event) {
```

Comment:
Why revert this part (from this [commit](https://github.com/dosbox-staging/dosbox-staging/pull/4129/commits/5653d2ad1586d43bfc4fa3e36d7e6e8103a031ea))? What's the connection with the stated purpose of the PR?

Will `event` ever be null?

Reply:
Well, I just reverted all commits from that PR. Check out the commit message of this commit in the linked PR for an explanation.

Reply:
That's fair, but I was just curious why that `event` null check early exit was needed and whether removing it re-adds some bug. That being said, such an analysis is certainly not a blocker for a revert of a known regression.

Reply:
Seems like it was needed for the equality checks. But it's the mapper, so anything goes 😅